packages/shared-state-pirania/files/etc/pirania/hooks/db_change/02-sync_db

```diff
@@ -1,2 +1,2 @@
 #!/bin/sh
-shared-state sync pirania
+shared-state sync pirania-vouchers
```

Comment:
The hooks folder should be documented within it, and outside of it somewhere, so people wanting to modify the behaviour can tell when that code is being executed or when code should be added here.

Reply:
have you seen this @spiccinini ?

Reply:
Hi Nico, yes. Indeed it should be Documented! This PR is not to fix every missing thing in Pirania, that is why I merged the branch even if not all improvement suggestions are done.